### PR TITLE
ER-870 close vacancy

### PR DESCRIPTION
**Pull request by shomavg**
Mainly copy paste from Employer!

### CODE REVIEW

File: src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs [hunk 1]
```diff
@@ -183,18 +180,15 @@ public async Task WhenCustomDraftSkillsHaveBeenAdded_ShouldBeOrderedByTheirPrefi
 
         private static Vacancy GetTestVacancy()
         {
-            return new Vacancy
-            {
+            return new Vacancy {
```
Comment (chrisjensenuk) [inline src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs]: Can we revert these formatting changes please.

File: src/Provider/UnitTests/Provider.Web/UtilityTests/CheckRouteIsValidForVacancyTests.cs [hunk 2]
```diff
@@ -17,8 +17,7 @@ public class CheckRouteIsValidForVacancyTests
         [InlineData("any other route", true)]
         public void ShouldRedirectToTitle(string route, bool shouldRedirect)
         {
-            var vacancy = new Vacancy
-            {
+            var vacancy = new Vacancy {
```
Comment (chrisjensenuk) [inline src/Provider/UnitTests/Provider.Web/UtilityTests/CheckRouteIsValidForVacancyTests.cs]: ditto

File: src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs (hunk 1 above)
Comment (shomavg) [reply src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs]: i think my resharper is the culprit!

File: src/Provider/Provider.Web/Views/CloseVacancy/Close.cshtml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        </h1>
+
+        <h2 class="govuk-heading-m govuk-!-margin-top-0">Closing a vacancy</h2>
+        <p class="govuk-body">By continuing with this action you will be closing this vacancy to further applications on Find an apprenticeship.</p>
+        <p class="govuk-body">It may take up to an hour to take effect on Find an apprenticeship.</p>
+        <div class="govuk-inset-text govuk-body">This action cannot be undone.</div>
+        <form asp-route="@RouteNames.CloseVacancy_Post">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--m govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+                <h3 class="govuk-fieldset__heading">
+                    Do you want to create a new vacancy or clone an existing vacancy?
```
Comment (sachpatel) [inline src/Provider/Provider.Web/Views/CloseVacancy/Close.cshtml]: We cannot ask them to clone a vacancy on the close vacancy page!

File: src/Provider/UnitTests/Provider.Web/UtilityTests/CheckRouteIsValidForVacancyTests.cs [hunk 4]
```diff
@@ -7,7 +7,7 @@
 using FluentAssertions;
 using Xunit;
 
-namespace Esfa.Recruit.UnitTests.Provider.Web.UtilityTests
+namespace Esfa.Recruit.Provider.UnitTests.Provider.Web.UtilityTests
```
Comment (sachpatel) [inline src/Provider/UnitTests/Provider.Web/UtilityTests/CheckRouteIsValidForVacancyTests.cs]: Why do we have to have Provider twice. The namespace should be `Esfa.Recruit.UnitTests.Provider.Web.UtilityTests`

This is not consistent but this is what we need to start doing, have UnitTests at the higher level in the namespace and component breakdown after.

File: src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs (hunk 1 above)
Comment (sachpatel) [reply src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs]: Stop using resharper

Comment (shomavg) [reply src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs]: Resharper is innocent!
https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave - this is formatting.

File: src/Provider/Provider.Web/Views/CloseVacancy/Close.cshtml (hunk 3 above)
Comment (shomavg) [reply src/Provider/Provider.Web/Views/CloseVacancy/Close.cshtml]: Good spot, Sachin!

**Pull request merged**